tests/Query/QueryDefaultFunctionsTest.php

```diff
@@ -16,6 +17,13 @@
  */
 class QueryDefaultFunctionsTest extends \PHPUnit\Framework\TestCase
 {
+	public const TMP = KIRBY_TMP_DIR . '/Query.QueryDefaultFunctions';
+
+	public function tearDown(): void
+	{
+		Dir::remove(static::TMP);
+	}
```

Comment:
Question for understanding: why do the individual test classes have to take care of removing the tmp folder, but not creating it?

Reply:
It depends. Some test classes do create the tmp folder because the tests only write files to it. But other classes like this one use the tmp folder as the index root of an app instance. The app automatically creates the folder if it doesn't exist. That's why creating is not needed here. But it still needs to be removed for the following tests in the same class and for general cleanup.